aml-development/frontend
# feat(routing): initial react router with redux

AMLOS-583

**Description**
Implement routing on the AML 3.0 front end

**Acceptance Criteria**
Routing is available in the AML 3.0 application.

**How to Test**
Pull this branch

Check that storefront loads on empty path ("/#/" or "/")
Click on a listing tile.  Check that dummy listing page loads with url path "/#/listing/<id>".  Manually typing the URL should work as well.
Check that other url paths ("/#/asdf") return blank (TODO: add React 404 page?)
Check browser history (clicking back and forward) reloads pages appropriately
